Updated Abstract Section (#43)
greenelab/knowledge-graph-review: content/01.abstract.md

Knowledge graphs can support many biomedical applications.
These graphs represent biomedical concepts and relationships in the form of nodes and edges.
In this review, we discuss how these graphs are constructed and applied with a particular focus on how machine learning approaches are changing these processes.

> In many examples in the literature, biomedical knowledge graphs have been constructed from pre-existing databases that are populated by experts via manual curation, but we are now seeing a more robust use of automatic systems.

Reviewer: ```suggestion
Biomedical knowledge graphs have often been constructed by integrating databases that were populated by experts via manual curation, but we are now seeing a more robust use of automated systems.
```
not sure on this - optional revision - your call

A number of techniques are used to represent knowledge graphs, but often machine learning methods are used to construct a low-dimensional representation that can support many different applications.
This representation is designed to preserve a knowledge graph’s local and/or global structure.
Additional machine learning methods can be applied to this representation to make predictions within genomic, pharmaceutical, and clinical domains.

> We frame our discussion first around knowledge graph construction and then around unifying relational learning techniques as well as unifying applications separately.

Reviewer: ```suggestion
We frame our discussion first around knowledge graph construction and then around unifying relational learning techniques and unifying applications.
```
